README.md

```diff
@@ -1 +1,263 @@
+</div>
+```
+
+If we were to then update `$scope.twitter` in `SomeController`, we'd end up with:
+
+```html
+<div ng-controller="SomeController">
+	new value!
+	<twitter-card>
+		<div class="twitter">
+			<a href="https://twitter.com/billgates">Follow @billgates on Twitter!</a>
+		</div>
+	</twitter-card>
+</div>
+```
+
+The Twitter handle inside `<twitter-card>` hasn't changed. This is because we've created a new scope (initially based off of the parent scope) - they are no longer linked!
+
+### scope: {}
+
+Now, this is where things get funky. We can pass through an object to our scope object - but why?
+
+When we use normal HTML elements, such as `<input />`, we configure it via attributes. For instance, we might give it a name - we'd do this like such:
+
+```html
+<input name="ourInputName" />
+```
+
+Hold on a minute.. Can we configure our directives by passing through attributes?
+
+Yes we can!
+
+To do this, we need to specify *what* attributes we want to be configured by. Using our Twitter card above, the only thing we need to configure it is the Twitter handle.
+
+To grab this value out of the attributes, we can specify the attribute name in the object. For instance, if we wanted to utilise the Twitter card as such:
+
+```html
+<twitter-card handle="billgates"></twitter-card>
+<twitter-card handle="bob"></twitter-card>
+```
+
+We'd put the property `handle` into our scope object.
+
+Now, what do we put as the value? There are two that we can use:
+
+#### @ (at)
+
+If we pass through `@` as the value, as so:
+
+```js
+function TwitterCard() {
+	return {
+		template: [
+			'<div class="twitter">',
+				'<a href="https://twitter.com/{{ handle }}">Follow @{{ handle }} on Twitter!</a>',
+			'</div>'
+		].join(''),
+		scope: {
+			handle: '@'
+		},
+		restrict: 'E'
+	};
+}
+
+angular
+	.module('app')
+	.directive('twitterCard', TwitterCard);
+```
+
+What `@` tells Angular is to just copy the value as it is. This will literally copy whatever we put in the `handle` attribute and put it into our scope.
+
+#### = (equals)
+
+However, if we want to be able to receive variables through, we can use `=`.
+
+```js
+function TwitterCard() {
+	return {
+		template: [
+			'<div class="twitter">',
+				'<a href="https://twitter.com/{{ handle }}">Follow @{{ handle }} on Twitter!</a>',
+			'</div>'
+		].join(''),
+		scope: {
+			handle: '='
+		},
+		restrict: 'E'
+	};
+}
+
+angular
+	.module('app')
+	.directive('twitterCard', TwitterCard);
+```
+
+What this means is that if we have a controller with two properties on it, such as `twitterHandle1` and `twitterHandle2`, we can do this:
+
+```html
+<twitter-card handle="twitterHandle1"></twitter-card>
+<twitter-card handle="twitterHandle2"></twitter-card>
+```
+
+Angular will then change the values over to their actual scope values before passing the value through to our directive. This also means that if we were to update either of them, Angular will automatically update our directive to match it too. Awesome!
```

Comment:
Is this generally what you want to do ? I can't imagine a scenario where I wouldn't want my directive scope to update along with my controller, right?